src/services/rfqm_service.ts

```diff
@@ -51,6 +51,7 @@ export enum RfqmTypes {
 
 export interface FetchIndicativeQuoteParams {
     apiKey: string;
```

Comment:
might be worth leaving a TODO on this field (`apiKey`) as well to remove in the future

Reply:
gonna try to take it out now

Reply:
took it out